apps/frontend/components/FormValidation/IncidentFormValidation/IncidentFormValidation.tsx

```diff
@@ -49,8 +46,8 @@ export const IncidentFormValidation = ({
         district: formattedForm.district,
         commune: formattedForm.commune,
       },
-      interviewer: formattedForm.reportName,
-      disTyp: formattedForm.disasterType,
+      interviewer: formattedForm.entryName,
```

Comment:
There seems to be quite a bit of duplication between the different FormValidation.tsx files